src/EditorFeatures/CSharp/EventHookup/EventHookupCommandHandler_SessionCancellingCommands.cs

```diff
@@ -19,7 +18,7 @@ internal partial class EventHookupCommandHandler :
     public bool ExecuteCommand(EscapeKeyCommandArgs args, CommandExecutionContext context)
     {
         _threadingContext.ThrowIfNotOnUIThread();
-        EventHookupSessionManager.CancelAndDismissExistingSessions();
+        EventHookupSessionManager.DismissExistingSessions();
```

Comment:
Dismiss inherently will cancel, if the work has not been detached from teh session.